src/components/carousel.js

```diff
@@ -0,0 +1,48 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import MovieCard from "./movie-card/movie-card";
+import SwiperCore, { Pagination, Navigation } from "swiper";
+SwiperCore.use([Pagination]);
+SwiperCore.use([Navigation]);
+
+export function Carousel(items) {
+  console.log(items);
+  return (
+    <>
+      <Swiper
+        navigation={true}
```

Comment:
Lets define this as a prop with a default value in the `Carousel` component, same goes for all the below props, it would look something like this:
```js
const defaultBreakpoints = {
  375: { ... },
  640: { ... },
  768: { ... },
  1247: { ... },
}

export function Carousel({
  items = [],
  navigation = true,
  spaceBetween = 30,
  slidesPerView = 5,
  pagination = false, // let's check if it works like this
  breakpoints = defaultBreakpoints,
}) {
  return (
    ...
    <Swiper
      navigation={navigation}
      spaceBetween={spaceBetween}
      slidesPerView={slidesPerView}
      ...etc
    >
    ...
  );
}
```